pkg/environment/api/project.go

```diff
@@ -120,8 +120,12 @@ func (s *EnvironmentService) ListProjects(
 		return nil, err
 	}
 	whereParts := []mysql.WherePart{}
-	if req.OrganizationId != "" {
-		whereParts = append(whereParts, mysql.NewFilter("organization_id", "=", req.OrganizationId))
+	if req.OrganizationIds != nil && len(req.OrganizationIds) > 0 {
```

Comment:
The `len` also checks for nil.

```suggestion
	if len(req.OrganizationIds) > 0 {
```